WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -458,6 +502,25 @@ private extension OrderDetailsDataSource {
                                                     comment: "VoiceOver accessibility label for the shipping notice about the order")
     }
 
+    private func configureInstallWCShip(cell: WCShipInstallTableViewCell) {
+        cell.update(title: NSLocalizedString("Need a shipping label?",
+                                             comment: "Title of the banner in the Order Detail for suggesting to install WCShip extension."),
+                    body: NSLocalizedString("Print labels from your phone, with WooCommerce Shipping.",
+                                            comment: "Body of the banner in the Order Detail for suggesting to install WCShip extension."),
+                    action: NSLocalizedString("Get WooCommerce Shipping",
+                                              comment: "Action of the banner in the Order Detail for suggesting to install WCShip extension."),
+                    image: .installWCShipImage)
+
+        cell.selectionStyle = .none
+
+        cell.accessibilityTraits = .staticText
```

Comment:
Should we use `.button` instead? 🤔 
Perhaps we can add a TODO for now and change it after adding the functionality.

Reply:
Good catch 👍 updated